Python/specialize.c

```diff
@@ -2466,6 +2510,12 @@ LONG_FLOAT_ACTION(compactlong_float_multiply, *)
 LONG_FLOAT_ACTION(compactlong_float_true_div, /)
 #undef LONG_FLOAT_ACTION
 
+static _PyBinaryOpSpecializationDescr compactlongs_specs[NB_OPARG_LAST+1] = {
```

Comment:
I think these don't hold any mutable data as such can be marked `static const`.

Reply:
I tried that and the compiler complained in a couple of the buildbots.